openxr/examples/vulkan.rs

```diff
@@ -37,21 +40,32 @@ fn main() {
     let entry = xr::Entry::load()
         .expect("couldn't find the OpenXR loader; try enabling the \"static\" feature");
 
+    #[cfg(target_os = "android")]
+    entry.initialize_android_loader().unwrap();
+
     // OpenXR will fail to initialize if we ask for an extension that OpenXR can't provide! So we
     // need to check all our extensions before initializing OpenXR with them. Note that even if the
     // extension is present, it's still possible you may not be able to use it. For example: the
     // hand tracking extension may be present, but the hand sensor might not be plugged in or turned
     // on. There are often additional checks that should be made before using certain features!
     let available_extensions = entry.enumerate_extensions().unwrap();
 
+    dbg!(&available_extensions);
+
     // If a required extension isn't present, you want to ditch out here! It's possible something
     // like your rendering API might not be provided by the active runtime. APIs like OpenGL don't
     // have universal support.
-    assert!(available_extensions.khr_vulkan_enable2);
+    assert!(available_extensions.khr_vulkan_enable || available_extensions.khr_vulkan_enable2);
 
     // Initialize OpenXR with the extensions we've found!
     let mut enabled_extensions = xr::ExtensionSet::default();
-    enabled_extensions.khr_vulkan_enable2 = true;
+    enabled_extensions.khr_vulkan_enable = available_extensions.khr_vulkan_enable;
```

Comment:
We should not enable version 1 if version 2 is available. Additionally, a comment should be added documenting that the fallback path is a temporary compatibility measure for Oculus runtimes.